src/errors/test-run/templates.js

```diff
@@ -108,6 +120,30 @@ export default {
         ${err.getCallsiteMarkup()}
     `),
 
+    [TYPE.actionElementNonEditableError]: err => markup(err, `
+        The action element is expected to be editable (an input, textarea or element with the contentEditable attribute).
+
+        ${err.getCallsiteMarkup()}
+    `),
+
+    [TYPE.actionElementNonContentEditableError]: err => markup(err, `
+        The action element is expected to have the contentEditable attribute enabled.
```

Comment:
Also may be we should mention about it in selectEditableContent command description in future